sample_1.py

```diff
@@ -0,0 +1,16 @@
+def fib(n) :
```

Comment:
#function name can be calculateFibonacciSum

Reply:
Great input type checking. I've added isinstance(n, int) to check the data class and return 0 if negative.